gr-digital/python/digital/utils/lfsr.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 Free Software Foundation, Inc.
+# 
+# This file is part of GNU Radio
+# 
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# 
+
+def lfsr_args(seed,*exp):
```

Comment:
volk_profile was run, and I compiled and test that both variants work, that is, both volk_popcnt and builtin. Volk_profile was run, and it says:
volk_64u_popcnt a_sse4_2 generic
volk_64u_popcnt generic generic
volk_64u_popcnt a_sse4_2 generic

Not sure why i get 3 of them.

The volk_popcnt is pretty much the same as the original.

I tried implementing the builtin_parityl-call as a kernel i volk but what happens in volk seems to eat up any performance gain, sadly. 

About 32-bit, the builtin_parityl seems to solve this issue for us, but someone with access to a system that can test this would answer any doubts.

As for lfrs_args, I agree to the signature-change, however my suggestion is to leave it as for now, that lfrs_args is just there now for the QA. Let's do a makeover regarding all gr-digital blocks in another commit?